### PR TITLE
chore(readme): fix typo

### PR DESCRIPTION
**Pull request by millsp**

### CODE REVIEW

**Review by ocdtrekkie — APPROVED** [review]
Glad I checked the PR tab before opening my own!

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ As of this writing, some major features are missing which we intend to fix short
...
 
-`workerd` tries to isolate each Worker so that it can only access the resources it is configured to access. However, `workerd` on its own does not contain suitable defense-in-depth against the possibliity of implementation bugs. When using `workerd` to run possibly-malicious code, you must run it inside an appropriate secure sandbox, such as a virtual machine. The Cloudflare Workers hosting service in particular [uses many additional layers of defense-in-depth](https://blog.cloudflare.com/mitigating-spectre-and-other-security-threats-the-cloudflare-workers-security-model/).
+`workerd` tries to isolate each Worker so that it can only access the resources it is configured to access. However, `workerd` on its own does not contain suitable defense-in-depth against the possiblity of implementation bugs. When using `workerd` to run possibly-malicious code, you must run it inside an appropriate secure sandbox, such as a virtual machine. The Cloudflare Workers hosting service in particular [uses many additional layers of defense-in-depth](https://blog.cloudflare.com/mitigating-spectre-and-other-security-threats-the-cloudflare-workers-security-model/).
```
[the remark this answers is not in the document]
Comment (ocdtrekkie) [reply README.md:54]: Heh, wow, I failed to check the typo fix for a typo. Nice catch, @jclee

**Review by ocdtrekkie — CHANGES_REQUESTED** [review]
Unapprove, see jclee's comment

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (millsp) [reply README.md:54]: :see_no_evil: Thanks

**Review by jasnell — APPROVED** [review]

**Comment by kentonv** [comment]
Thanks!

**Pull request merged**